pkgs/os-specific/darwin/apple-source-releases/Libsystem/default.nix

```diff
@@ -12,22 +12,11 @@ let
     url = "https://github.com/darlinghq/darling/archive/d2cc5fa748003aaa70ad4180fff0a9a85dc65e9b.tar.gz";
     sha256 = "11b51fw47nl505h63bgx5kqiyhf3glhp1q6jkpb6nqfislnzzkrf";
```

Comment:
Please also update the hash.

Reply:
Whoops, sorry. Things went a bit fast. Done.